src/dotnet/UI.Blazor.App/Components/AddMember/AddMemberModalPage.razor

```diff
@@ -8,11 +8,27 @@
         ListSource="@_backend"
         Changed="OnSelectionChanged" />
 </FormBlock>
+@if (Context.Model is PlaceId && !string.IsNullOrEmpty(_placeLink)) {
```

Comment:
It's enough to check single condition:
`!string.IsNullOrEmpty(_placeLink)`
or
`_shareRequest is not null`